api/src/main/java/org/openmrs/util/ThreadSafeCircularFifoQueue.java

```diff
@@ -756,15 +756,12 @@ public void remove() {
 				
 				if (prevIndex == NONE) {
 					throw new IllegalStateException();
-				} else if (prevIndex >= 0) {
-					if (elements[prevIndex] == prevItem) {
+				} else if ((prevIndex >= 0) && (elements[prevIndex] == prevItem)) {
 						internalRemoveAtIndex(prevIndex);
-
-						if (prevIndex != read) {
+			      	} else if (prevIndex != read) {
 							nextIndex = Math.max(prevIndex, read);
-						}
-					}
-				}
+		    	}
+				
```

Comment:
![image](https://user-images.githubusercontent.com/9057882/91222260-38baf380-e71f-11ea-9e5c-00784891ef7e.png)

Reply:
Have you tried and see how tha full code ends up with whole code format. To me reverting that lines will make the code to look wierd